data/Language/CoffeeScript.yaml

```diff
@@ -0,0 +1,64 @@
+identifier: CoffeeScript
+wikidata: Q1106819
+creation_date: 20091213
+extensions:
+  - coffee
+  - litcoffee
+line_continuation:
+  - backslash
+delimiters:
+  - hash
+  - triple_hash
+  - single_quote_slash_escape
+  - double_quote_slash_escape
+  - triple_single_quote
+  - triple_double_quote
+  - backticks
+  - curly_braces
+  - round_braces
+  - square_braces
+datatypes:
+  - bool_lower_case
```

Comment:
`bool_lower_case` doesnt exist yet. Besides, You will need to create the datatype definition for CoffeeScript, because it is not like other languages datatype.